### PR TITLE
Hacktoberfest blog post

### PR DESCRIPTION
**Pull request by hexsecs**
# Summary of Changes

Created a blog article to discuss our work with Hacktoberfest. Improvements to the text welcome.

## Checklist

- [x] I have read and followed the [project's contribution guidelines](https://github.com/secure-systems-lab/lab-guidelines/blob/master/dev-workflow.md), including code style and commit message conventions.
- [x] My code is well-documented, and I've updated relevant documentation.
- [x] I have added or updated test cases to ensure the code's functionality.
- [x] I have tested these changes on my local environment.
- [x] All tests pass, and there are no new linting errors.
- [x] I have reviewed and proofread my code and the changes.
- [x] The branch is up-to-date with the base branch.

## Screenshots (if applicable)

image

## Additional Context

Add any additional context or information that might be helpful for reviewers.

## Reviewer(s)

@jhdalek55 @Abhijay007

### CODE REVIEW

**Review by Abhijay007 — APPROVED** [review]
LGTM, we plan to further categorize the blogs in the near future. Once we have a sufficient number of blogs, for example, we might require at least four blogs to create meaningful categories. I'll investigate the options and customizations available for blogs using Docusaurus.

File: blog/2023-10-28-hacktoberfest/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+authors:
+  - Phil
+---
+
+![](hf10_banner_1032x600.png)
+As the leaves change color 🍂 and the breeze carries a hint of chill, the open-source community gears up for a month of code, collaboration, and coffee-fueled late nights ☕. [Hacktoberfest](https://hacktoberfest.com/), the annual open-source celebration, is here again, and this year, Uptane is joining the fray!
+
+Uptane, a robust framework designed to secure software updates for automobiles 🚗, is participating in Hacktoberfest 2023 to foster community engagement and encourage contributions to its project. With cybersecurity becoming a paramount concern in the automotive industry, the initiative taken by Uptane is a timely and significant one.
+
+This Hacktoberfest, developers from around the globe 🌍 can contribute to Uptane's repository, helping to enhance the security framework and make our vehicles more secure. Whether you're a seasoned developer or a newbie looking to make your first open-source contribution, Uptane welcomes you with open arms 🤗.
```
Comment (jhdalek55) [inline blog/2023-10-28-hacktoberfest/index.md:12]: One small fix here. Take out the word "our" in the first sentence. Uptane doesn't create or own the vehicles.
```suggestion
This Hacktoberfest, developers from around the globe 🌍 can contribute to Uptane's repository, helping to enhance the security framework and make vehicles more secure. Whether you're a seasoned developer or a newbie looking to make your first open-source contribution, Uptane welcomes you with open arms 🤗.
```

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
I'm requesting one tiny deletion. Otherwise, this looks great. I suggest we cross post this on the new LinkedIn page as well.

**Comment by hexsecs** [comment]
> I'm requesting one tiny deletion. Otherwise, this looks great. I suggest we cross post this on the new LinkedIn page as well.

@jhdalek55 - I incorporated your change request and merged the pull request.

**Pull request merged**